catalog/Data_Analysis/structures.yml

```diff
@@ -1,7 +1,5 @@
 name: Structures
 description: Projects for working with data structures such as vectors and data frames.
 projects:
-  - daru
   - daru-io
-  - daru-view
```

Comment:
This is the view component of the DARu framework and [relocated in Visualization](https://github.com/rubytoolbox/catalog/pull/527/files#diff-54e6f83e3b5c393c0ec484221bbd50b690b53bb1e182f10f1f022817a1a0e3c2R4)